www: Various documentation improvements (#1635)
denoland/fresh: docs/latest/concepts/routing.md

// ...
```

## Route Groups

> When working with [route layouts](/docs/canary/concepts/route-layout) or
> [route middlewares](/docs/canary/concepts/middleware) you'll sometimes come
> across a situation where you want your routes to inherit from a different layout
> than another route that's in the same URL segment.

Reviewer: ```suggestion
When working with [route layouts](/docs/canary/concepts/route-layout) or
[route middlewares](/docs/canary/concepts/middleware), you'll sometimes come
across a situation where you want your routes to inherit from a layout
other than what's suggested by the URL segment.
```

Author: Thanks!
